### PR TITLE
Use minor version as Drupal constraint

### LINKED ISSUE #19: Use Minor version for installing Drupal
Instead of specific patch version, use minor version constraint for Composer when installing Drupal.

### PR DESCRIPTION
**Pull request by back-2-95**
Instead of installing a specific patch version of Drupal, install latest minor version.

Will resolve #19

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,14 +12,14 @@ COPY --from=drupal /usr/local/etc/php/conf.d/* /usr/local/etc/php/conf.d/
 
 COPY --from=composer/composer:2-bin /composer /usr/local/bin/
 
-# https://www.drupal.org/project/drupal/releases/10.1.2
-ENV DRUPAL_VERSION 10.1.2
+# https://www.drupal.org/project/drupal/releases
+ENV DRUPAL_VERSION 10.1
 
 WORKDIR /opt/drupal
 # Copy from the official image when Drupal 10.0 will be released
```
Comment (dunglas) [inline Dockerfile:19]: Cannot we do this instead now?

File: docker-compose.yml [hunk 2]
```diff
@@ -9,8 +9,6 @@
 # Database password: example
 # ADVANCED OPTIONS; Database host: postgres
 
-version: '3.1'
-
 services:
```
Comment (dunglas) [inline docker-compose.yml:12]: We should also rename the file `compose.yaml` to be in sync with be Docker best practices.

**Review by dunglas — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (back-2-95) [reply Dockerfile:19]: I really don't remember what this refers to 😂 
Not sure if Drupal releases any official Docker images.

File: docker-compose.yml (hunk 2 above)
Comment (back-2-95) [reply docker-compose.yml:12]: Can do, is there some documentation for this? This is a new thing to me.

Comment (dunglas) [reply docker-compose.yml:12]: Yes: https://docs.docker.com/compose/compose-file/03-compose-file/

Just renaming the file should be enough.

File: Dockerfile (hunk 1 above)
Comment (dunglas) [reply Dockerfile:19]: The official images are here: https://hub.docker.com/_/drupal

Something like `COPY --from=drupal ---link /opt/drupal /opt/drupal` should do the trick and be always up to date.

(https://github.com/docker-library/drupal/blob/master/Dockerfile.template)

Comment (back-2-95) [reply Dockerfile:19]: I'll check this

File: docker-compose.yml (hunk 2 above)
Comment (back-2-95) [reply docker-compose.yml:12]: Hmm, for me any `docker compose command` will result with `no configuration file provided: not found` after renaming the file. Docker Compose version v2.22.0

**Comment by dunglas** [comment]
Thank you for your contribution @back-2-95.
By the way, would you like to become co-maintainer of this repo?

**Comment by back-2-95** [comment]
@dunglas hi, I can become co-maintainer. This is quite simple repo. That said, I'd like to set some goals:

Crystallising the purpose:
- PoC or template for a project
- Now it's more like PoC in a sense that it would need manual steps and changes to be used in a real project imho

Providing some quickstart for Drupal community:
- like DDEV setup and maybe some others
- this would help Drupal community members advance their issues regarding FrankenPHP

FYI: I don't actively code Drupal sites myself, I mostly provide tooling and devops for Drupalists for their projects.

**Pull request closed without merging**